docs/adr/0004-naming-convention-images.md

```diff
@@ -0,0 +1,42 @@
+# Project Thoth Naming Convention Schema for Images
+
+* Status: proposed
+* Date: 2021-Jun-17
+
+Technical Story: As Thoth goal to provide curated stack and images, it would be nice to have a convention for naming of the images.
+
+## Context and Problem Statement
+
+Image names are important for branding and let others identify easily a specific image they need. For example "I want to work on computer vision project with Tensorflow, what stack and image should I use?" Having a trusted well maintained source of images with clean naming convention can help on that.
+
+## Considered Options
+
+* `s2i-{application}` standard name currently, but not everyone knows what S2I is.
+* `odh-{application}` for branding/marketing having ODH in front seems to be the best solution, in this way the name will be shorter as well.
+* `curated-stack-{application}` as it shows what our intention is: we want to provide a curated/predictable software stack, it might be used by ODH or RHODS or others, it might use S2I or other technology.
```

Comment:
during our 'tech talk' discussion there was an argument that the length of the repo name on quay should be considered, so let's go with `ps` and the `predictable stack`

Reply:
Perfect! If all good we can merge this PR then and start creating the new repos!